src/app/routes/assets/verification-method/verification-method.component.ts

```diff
@@ -30,7 +31,8 @@ export class VerificationMethodComponent implements OnInit {
 
   constructor(private dialogRef: MatDialogRef<VerificationMethodComponent>,
               @Inject(MAT_DIALOG_DATA) private dialogData: any,
-              private verificationService: VerificationService) {
+              private verificationService: VerificationService,
+              private toastr: SwitchboardToastrService) {
```

Comment:
By adding toastr you need to add provider in test. That way your tests stop failing.